wisecondor.py

```diff
@@ -453,6 +452,10 @@ def main():
 	parser_test.add_argument('-minzscore',
 		type=float, default=None,
 		help='Minimum absolute z-score')
+	parser_test.add_argument('-chromosomes', 
+	help="Integer of every chromosome to plot", 
+	type=(lambda x: map(int,x.split(','))),
+	default=range(1,23) ),
```

Comment:
This might be githubs markup but I think for the sake of consistency lines 456, 457 and 458 need an additional indent.